packages/dashboard/src/components/dashboard/index.tsx

```diff
@@ -40,17 +41,19 @@ const Dashboard: React.FC<DashboardProperties> = ({
     <ClientContext.Provider value={getClients(clientConfiguration)}>
       <QueryContext.Provider value={getQueries(clientConfiguration)}>
         <QueryClientProvider client={queryClient}>
-          <Provider store={configureDashboardStore({ ...toDashboardState(dashboardConfiguration), readOnly })}>
-            <DndProvider
-              backend={TouchBackend}
-              options={{
-                enableMouseEvents: true,
-                enableKeyboardEvents: true,
-              }}
-            >
-              <InternalDashboard onSave={onSave} editable={true} />
-            </DndProvider>
-          </Provider>
+          <AppKitConfig>
+            <Provider store={configureDashboardStore({ ...toDashboardState(dashboardConfiguration), readOnly })}>
```

Comment:
curious: why do use context instead of using some state management ?

Reply:
also considered state management but most of the app is context based so i mostly didnt want to throw off the current way of doing things.

i dont know the History of context vs state management but maybe someone that has been here longer has more info on that !